jpo-ode-core/pom.xml

```diff
@@ -85,6 +85,12 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+		<groupId>commons-io</groupId>
+		<artifactId>commons-io</artifactId>
+		<version>2.1</version>
```

Comment:
Overriding managed version 2.4 for commons-io. Please remove version and scope lines.